caffe_app/views/import_prototxt.py

```diff
@@ -1,26 +1,32 @@
 from django.shortcuts import render
 from django.views.decorators.csrf import csrf_exempt
 from django.http import JsonResponse
+from django.conf import settings
 import yaml
+import os
+#BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
@utsavgarg Please remove the comment. There is no need to comment this. We can always recover this. :)